Assets/UI/Scripts/PanelPager.cs

```diff
@@ -51,11 +58,17 @@ void Start()
     /// </summary>
     private void SetButtonsInteractable()
     {
+        // Sets the previous button active on every panel except the first
         if (PreviousButton) {
-            PreviousButton.interactable = currentIndex > 0;
+            PreviousButton.gameObject.SetActive(currentIndex > 0);
         }
+        // Sets the next button active on every panel except the last
         if (NextButton) {
-            NextButton.interactable = currentIndex < Panels.Count - 1;
+            NextButton.gameObject.SetActive(currentIndex < Panels.Count - 1);
+        }
+        // Only sets the done button active on the last panel
+        if (DoneButton) {
```

Comment:
Instead of having a separate "Done" button, could the `Text` of the next button just be changed to "Done" for the last panel?

Reply:
I thought about that, but then we'd have to have two script components for that button (one for next functionality and one for exit functionality) and figure out a way to choose which one to use depending on which panel was active, which seemed like more work.

Reply:
That makes a lot of sense, let's stick with your way!